uber/site_sections/accounts.py

```diff
@@ -19,7 +19,7 @@ def index(self, session, message=''):
             'all_attendees': sorted([
                 (id, '{} - {}{}'.format(name.title(), BADGES[badge_type], ' #{}'.format(badge_num) if badge_num else ''))
                 for id, name, badge_type, badge_num in session.query(Attendee.id, Attendee.full_name, Attendee.badge_type, Attendee.badge_num)
-                                                              .filter(Attendee.email != '').all()
+                                                              .filter(Attendee.email != '').filter(Attendee.status != INVALID_STATUS).all()
```

Comment:
@earl7399 Can you add this and submit a PR for it? We only want NEW and COMPLETED attendees in the accounts list, actually. Don't forget that the column name changed to badge_status.